_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -6126,6 +6126,7 @@
 /area/maintenance/port/fore)
 "ank" = (
 /obj/machinery/door/airlock/maintenance{
+	abandoned = 1;
```

Comment:
Since the vast majority of these are /maintenance or /maintenance_hatch, you should make a /abandoned subtype of them and use that instead

Reply:
I'm gonna have to push back on this one. While most of these might fall under that category, there are several that don't and it would cause issues since my understanding is that many of these airlocks are already var-edited for departmental access. 

It would be a lot of trouble for a change that's only going to cause more trouble.

Reply:
@ShizCalev You're the map guy, thoughts?

Reply:
Actually, I don't really care

Reply:
Will take 10 seconds to do, I'll make a PR on your branch @Robustin

Reply:
@Robustin https://github.com/Robustin/tgstation/pull/4